src/css/profile/mobile/common/listview.less

```diff
@@ -188,7 +191,7 @@ tau-expandable {
 			&::after {
 				content: "";
 				position: absolute;
-				width: 1 * @px_base;
+				width: @px_base;
```

Comment:
For me this format makes confusion, eg.
`padding: 0 3 * @px_base @px_base 0`

Reply:
You mean 1 * @px_base looks better than @px_base?

Reply:
I mean, it's compatible with the general format:
`0` or `1px` ... `10px` 
`0` or `1 * @px_base` ... `10 * @px_base`

Reply:
Yes,, But, Do we use the @px_base for the scability?
Anyway,, I don't understand what I do.. :(

Reply:
we use @px_base to allow proper dp-->CSS PX convertion
currently 1 dp = 1 CSS PX
but there may be devices with different scale in future